doc/main.html

```diff
@@ -958,16 +958,23 @@ <h4>Tables</h4>
           <li>one or more <code>tbody</code> element, with <code>tr</code> element(s) as needed for each row</li>
         </ul>
 
-        <p class="note">Page breaks are avoided within a <code>tbody</code> element when generating PDF. Care should be taken for especially long tables to ensure rendering is as expected. For data in tables spanning multiple pages, other options should be considered and used.</p>
+        <p>Unless all columns are uniform in contents, the relative width of
+        each column should be specified by setting the CSS <code>width</code>
+        property as a percentage value on each <code>tr</code> element.</p>
```

Comment:
This says `tr` element, but the example shows on the `th` element. This should say `th`?